mountainsort/src/core/msprocessmanager.cpp

```diff
@@ -82,179 +89,212 @@ void MSProcessManager::loadDefaultProcessors()
     loadProcessor(new filter_events_Processor);
 }
 
-bool MSProcessManager::containsProcessor(const QString &processor_name) const
+bool MSProcessManager::containsProcessor(const QString& processor_name) const
 {
-	return d->m_processors.contains(processor_name);
+    return d->m_processors.contains(processor_name);
 }
 
-bool MSProcessManager::checkProcess(const QString &processor_name, const QMap<QString, QVariant> &parameters) const
+bool MSProcessManager::checkProcess(const QString& processor_name, const QMap<QString, QVariant>& parameters) const
 {
-	return d->find_processor(processor_name)->check(parameters);
+    return d->find_processor(processor_name)->check(parameters);
 }
 
-
-QString compute_process_code(const QString &processor_name,const QMap<QString,QVariant> &parameters) {
-	QMap<QString,QVariant> X;
-	X["processor_name"]=processor_name;
-	X["parameters"]=parameters;
-	QString json=toJSON(X);
-	return compute_hash(json);
+QString compute_process_code(const QString& processor_name, const QMap<QString, QVariant>& parameters)
+{
+    QMap<QString, QVariant> X;
```

Comment:
Use QVariantMap instead of QMap<QString,QVariant>.

Why is the value in the map QVariant anyway?